learn-array-methods-by-building-a-statistics-calculator/script.js

```diff
@@ -0,0 +1,42 @@
+const getMean = (array) => {
+  // teach the reduce method
+  const mean = array.reduce((acc, el) => acc + el, 0) / array.length;
```

Comment:
I'm sure you're already thinking along these lines, but since this is the first time learners will use the `reduce` method, it might be good to teach the longer form of the method where the accumulator is returned, then to refactor to this more concise version.